docs/draft/keycard.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Keyacrd? (have no idea how to call it)
+version: 0.0.0
+status: Draft
+authors: Roman Volosovskyi <roman@status.im>
+redirect_from:
+  - /keycard.html
+---
+
+## Table of Contents
+
+1. [Abstract](#abstract)
+2. [Definitions](#definitions)
+3. [Multiaccount creation](#Multiaccount-creationrestoring)
+4. [Multiaccount restoring via pairing](#Multiaccount-restoring-via-pairing)
+5. [Multiaccount unlocking](#Multiaccount-unlocking)
+6. [Where are the keys stored?](#Where-are-the-keys-stored)
+
+## Abstract
+
+In this specification, we describe how Satus communicates with Keycard to create, store and use multiaccount.
+
+## Definitions
+
+
+| Term | Description |
+| -------- | ------------------ |
+|      |      |
+
+
+## Multiaccount creation/restoring
+### Creation and restoring via mnemonic
```

Comment:
This seems to describe more how status-react is using status-go API. Is this accurate? If so, would it be possible to describe how status-go (e.g.) is interacting with Keycard?

Example: someone wants to write a new Status app in Android that interacts with Keycard and provides same capability as existing Status app

Reply:
I will add a separate section for `status-go` calls.

Reply:
@oskarth just to clarify, the doc describes how app interacts with `react-native-status-keycard` api, not `status-go`. But in order to use the card in the app we need to execute some status-go calls as well.

Reply:
https://github.com/status-im/react-native-status-keycard/blob/master/docs/usage.md